js/panel-manager.js

```diff
@@ -17,6 +17,12 @@ genotet.panelManager.TRANSITION_TIME = 300;
 genotet.panelManager.showPanel_ = true;
 
 /**
+ * Whether sends focused signal..
+ * @private {boolean}
+ */
+genotet.panelManager.sendSignal_ = false;
```

Comment:
True is used in view.js to avoid cyclic signal between focusing view and activating panel.